packages/playground/ssr-pug/index.pug

```diff
@@ -0,0 +1,8 @@
+doctype html
+html
+  head
+    meta(charset='UTF-8')
+    meta(name='viewport' content='width=device-width, initial-scale=1.0')
+    title SSR Pug
```

Comment:
The Pug test is identical to `ssr-html`, tests included, except for the obvious `.pug` extension.

Reply:
cc @patak-js Pug test added! 🙂 Open to any additions/changes you may have had in mind.